src/data/hassio/snapshot.ts

```diff
@@ -79,3 +79,21 @@ export const createHassioPartialSnapshot = async (
     data
   );
 };
+
+export const uploadSnapshot = async (
+  hass: HomeAssistant,
+  file: File
+): Promise<HassioSnapshot> => {
+  const fd = new FormData();
+  fd.append("file", file);
+  const resp = await hass.fetchWithAuth("/api/hassio/snapshots/new/upload", {
+    method: "POST",
+    body: fd,
+  });
+  if (resp.status === 413) {
```

Comment:
What is the limit here?

Reply:
1024MB

Reply:
OK, I guess that should be good enough as long as you don't add the media dir 😄